api/src/routing.js

```diff
@@ -95,7 +96,17 @@ if (process.argv.slice(2).includes('--allow-cors')) {
 
 app.use((req, res, next) => {
   req.id = uuid.v4();
-  next();
+  if (req.originalUrl === '/favicon.ico') {
+    pouch.medic.get('branding').then(doc => {
+      pouch.medic.getAttachment(doc._id, doc.resources.favicon).then(blob => {
+        res.send(blob);
+      });
+    }).catch(err => {
+      res.end(err);
+    });
+  } else {
+    next();
+  }
```

Comment:
This is not the right place to be handling this. When you see `app.use` like this it's middleware for doing things like checking every request for auth or whatever. What you want is a standard route handler, eg:

```js
app.get('/favicon.ico', function(req, res) {
  ...
});
```

These start around line 210 in the file.

You will probably have to remove the _other_ favicon from the `public` dir otherwise it'll take precedence.

Also we don't need two favicons so `git mv` the one from `api/src/public/favicon.ico` to the new location at `api/src/resources/ico/favicon.ico`.